test/rb_generate_data.py

```diff
@@ -0,0 +1,100 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+Code for generating data for the RB experiment for testing data analysis.
+"""
+
+import os
+import sys
+import json
+import qiskit_experiments as qe
+from qiskit.providers.aer import QasmSimulator
+from qiskit.providers.aer.noise import NoiseModel
+from qiskit.providers.aer.noise.errors.standard_errors import depolarizing_error
+
+
+def create_depolarizing_noise_model():
+    """
+    create noise model of depolarizing error
+    Returns:
+        NoiseModel: depolarizing error noise model
+    """
+    noise_model = NoiseModel()
+    p1q = 0.002
+    p2q = 0.01
+    noise_model.add_all_qubit_quantum_error(depolarizing_error(p1q, 1), "x")
+    noise_model.add_all_qubit_quantum_error(depolarizing_error(p1q, 1), "sx")
+    noise_model.add_all_qubit_quantum_error(depolarizing_error(p1q, 1), "rz ")
```

Comment:
rz has no noise
generally, I would suggest to look at one of the backends reported errors (you can do it using iqx) and provide realistic errors

Reply:
Changed the errors. the error parameters were took from "ibmq_manila" on 17 June 2021.